### DISCUSSION TITLE
Search in the _tabs does not work

### DISCUSSION THREAD

**Discussion by cx-8** [cotes2020/jekyll-theme-chirpy#1355] [Q&A] [answered]
### Checklist

- [X] I have read and accepted the [contributing guidelines](https://github.com/cotes2020/jekyll-theme-chirpy/blob/master/docs/CONTRIBUTING.md).

### How did you create the site?

Built from `jekyll-theme-chirpy`

### Description

Hi all!
Does anyone have any ideas on how to fix the search?
Search in the _tabs\ does not work

In the search we write the text that is in _tabs\about.md - no result.
Search only works by _posts\

### Operations you have already tried

-

**Comment by kungfux** [comment, marked as answer]
This is by design. The search includes data from [posts only](https://github.com/cotes2020/jekyll-theme-chirpy/blob/master/assets/js/data/search.json#L7).

**Reply by cx-8** [reply]
Hi! Thanks for your reply. Could you please tell me how I can replace tabs with posts on the left menu? What will the search function do?

**Reply by kungfux** [reply]
> how I can replace tabs with posts on the left menu

Each tab except `Home` is defined by a file in `_tabs` folder. You're free to add more files, rename/modify existing ones. Each file defines what layout will be used for it, what is the order in the side menu, and it's icon. Layouts are defined in `_layouts` folder.

```md
---
layout: tags
icon: fas fa-tags
order: 2
---
```

> What will the search function do?

It will continue to do what it is doing currently.

**Reply by cx-8** [reply]
Yes, that's right. But I want the search to be seen on tabs pages too!
